modules/nw-ingress-gateway-api-manage-succession.adoc

```diff
@@ -51,12 +58,19 @@ oc delete crd referencesgrants.gateway.networking.k8s.io
 +
 [IMPORTANT]
 ====
-Any controller that was previously managing the lifecycle of the Gateway API CRDs will fail to operate properly. Attempting to force its use in conjunction with the Ingress Operator to manage Gateway API CRDs might prevent the cluster update from succeeding.
+Deleting CRDs removes every custom resource that relies on them and can result in data loss. Back up any necessary data before deleting the Gateway API CRDs. Any controller that was previously managing the lifecycle of the Gateway API CRDs will fail to operate properly. Attempting to force its use in conjunction with the Ingress Operator to manage Gateway API CRDs might prevent the cluster update from succeeding.
 ====
 
 . Get the supported Gateway API CRDs by running the following command:
 +
 [source,terminal]
 ----
-$ oc apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.2.0/standard-install.yaml
+$ oc apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.2.1/standard-install.yaml
 ----
++
+[WARNING]
+====
+You can perform this step without deleting your CRDs. If your update to a CRD removes a field that is used by a custom resource, you can lose data. Updating a CRD a second time, to a version that re-adds a field, can cause any previously deleted data to reappear. Any third-party controller that depends on a specific Gateway API CRD version that is not supported in {product-title} {product-version} will break upon updating that CRD to one supported by Red{nbsp}Hat.
+====
```

Comment:
@Miciah I added this additional warning underneath the final step where the new CRDs get applied. wdyt?